pkgs/stdenv/generic/default.nix

```diff
@@ -152,10 +152,6 @@ let
         inherit lib config stdenv;
       }) mkDerivation;
 
-      # For convenience, bring in the library functions in lib/ so
-      # packages don't have to do that themselves.
-      inherit lib;
-
```

Comment:
external expressions galore will probably break by doing this. It will surely need a release note.
It would also be nice if we could have some sort of message to the user about it's removal, maybe a `throw`?

Reply:
I performed the removal to check if ofborg would error at all, and it seems like evaluation succeeded, so I'll revert it. Should we add a `trace` instead?

Reply:
OTOH if there is consensus to deprecate `stdenv.lib` entirely, it can be made into a `throw`.